teuthology/packaging.py

```diff
@@ -551,7 +551,7 @@ def base_url(self):
 
         For example::
 
-            http://gitbuilder.ceph.com/ceph-deb-raring-x86_64-basic/ref/master
+            http://gitbuilder.ceph.com/ceph-deb-raring-x86_64-default/ref/master
```

Comment:
Nit: gitbuilders went away years ago, replace with an appropriate shaman link or remove the example altogether.